Playbooks/AS-Revoke-Azure-AD-User-Session-From-Entity/azuredeploy.json

```diff
@@ -2,9 +2,9 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "metadata": {
-        "title": "AS-Revoke-Azure-AD-User-Session-From-Entity", 
+        "title": "AS-Revoke-Entra-ID-User-Session-From-Entity", 
         "description": "This playbook is intended to be run from a Microsoft Sentinel Entity. It will look up Azure AD users associated with the account entities and revoke their sessions.",
-        "prerequisites": "1. An App Registration with User.ReadWrite.All permissions granted on Microsoft Graph API. 2. A client secret for the App Registration must be generated. 3. An Azure Key Vault Secret to hold the client secret. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Revoke-Azure-AD-User-Session-From-Entity",
+        "prerequisites": "1. An App Registration with User.ReadWrite.All permissions granted on Microsoft Graph API. 2. A client secret for the App Registration must be generated. 3. An Azure Key Vault Secret to hold the client secret. Support for the set up and configuration of each of these items can be found here: https://github.com/Accelerynt-Security/AS-Revoke-Entra-ID-User-Session-From-Entity",
```

Comment:
Please replace Azure AD with Microsoft Entra ID

Reply:
@v-prasadboke this is fixed, please note https://github.com/Accelerynt-Security/AS-Revoke-Azure-AD-User-Session-From-Entity is left as AD so the link still works